neverwinter/nwscript/native/scriptcomplexical.cpp

```diff
@@ -1679,6 +1734,22 @@ int32_t CScriptCompiler::ParseNextCharacter(int32_t ch, int32_t chNext, const ch
 			return nReturnValue;
 		}
 	}
+	else if (m_nTokenStatus == CSCRIPTCOMPILER_TOKEN_BINARY_INTEGER)
```

Comment:
Can collapse into the above block with a couple of `||`s in the condition

Reply:
Went back and forth on this, but ended up keeping them separate for future modification and to make reading easier, but I'll throw them all together!  Fixed in 843bda77.